lib/freno/throttler.rb

```diff
@@ -187,9 +197,11 @@ def validate_args
       raise ArgumentError.new(errors.join("\n")) if errors.any?
     end
 
-    def all_stores_ok?(store_names)
+    def all_stores_ok?(store_names, **options)
+      check_kwargs = { app: app }
+      check_kwargs[:options] = options if options.any?
       store_names.all? do |store_name|
-        client.check?(app: app, store_name: store_name)
+        client.check?(store_name: store_name, **check_kwargs)
```

Comment:
Am I missing something here? Why have we moved `app` into `check_kwargs` but not `store_name`?

Given that we know `**options` will be a hash of some/no options, would it be simpler to change this to:

```ruby
client.check?(app: app, store_name: store_name, options: options)
```

Reply:
Yes we could! I was trying to avoid passing `options: {}` if we had no options to specify, allowing `Client#check?` to use its own default value for `options`. In this case, that default is `{}` though so it only makes a difference in principle! 😄

Reply:
Done!